index.html

```diff
@@ -1592,17 +1590,40 @@ <h3 id="applying">
         <div class="issue" data-number="446"></div>
       </section>
     </section>
-    <section>
+    <section data-dfn-for="webappmanifest" data-link-for="webappmanifest">
       <h2>
         Manifest and its members
       </h2>
       <p>
         A <dfn>manifest</dfn> is a JSON document that contains startup
         parameters and application defaults for when a web application is
-        launched. A manifest consists of a top-level <a>object</a> that
-        contains zero or more members. Each of the members are defined below,
-        as well as how their values are processed.
+        launched. A manifest consists of a top-level <dfn>WebAppManifest</dfn>
+        <a>object</a> that contains zero or more members.
```

Comment:
It's a dictionary, not an object.

Reply:
ok

Reply:
How do you want me to refer to that then. Object is defined as part of JSON, dictionary is not:

      <p>
        As the manifest uses the JSON format, this specification relies on the
        types defined in [[!ECMA-404]] specification: namely <dfn>object</dfn>,
        <dfn>array</dfn>, <dfn>number</dfn>, <dfn>string</dfn>,
        <code>true</code>, <code>false</code>, and <code>null</code>. Strict
        type checking is not enforced by this specification. Instead, each
        member's definition specifies the steps required to process a
        particular member and what to do when a type does not match what is
        expected.
      </p>